types/artifacts.d.ts

```diff
@@ -477,6 +477,12 @@ declare global {
         timestamp?: number;
       }
 
+      /* Used in cases where a metric may not be exposing a timing per-say, but still a numeric value */
+      export interface MetricValue {
```

Comment:
```suggestion
      export interface NonTimingMetric {
```

maybe?

Reply:
+1 on the name change, but I'd even personally prefer no named type at all for this, at least until more than one thing would use it (I think `interface Metric` only exists because we had more plans for it around audit-details we ended up not going through with).

`CumulativeLayoutShift.compute_` can just return `{Promise<{value: number, explanation?: string}>}`

Reply:
yeah don't want to encourage more of these nontiming metrics running around ;)